.github/workflows/tekton_bundle_push.yaml

```diff
@@ -17,21 +15,58 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: debianmaster/actions-k3s@master
-        id: k3s
-        with:
-          version: 'latest'
+      - run: |
+          mkdir -vp $HOME/{bin,.kube} || true
+          pushd $HOME/bin
+          curl -LO https://dl.k8s.io/release/v1.23.0/bin/linux/amd64/kubectl
+          popd
+          PATH="$HOME/bin:$PATH"
+          cat <<-EOF > $HOME/.kube/config
+          apiVersion: v1
+          clusters:
+          - cluster:
+              insecure-skip-tls-verify: true
+              server: https://api.example.com:6443
+            name: api-example-dot-com:6443
+          contexts:
+          - context:
+              cluster: api-example-dot-com:6443
+              namespace: default
+              user: kube:admin/api-example-dot-com:6443
+            name: default/api-example-dot-com:6443/kube:admin
+          current-context: default/api-example-dot-com:6443/kube:admin
+          kind: Config
+          EOF
       - uses: jerop/tkn@v0.1.0
       - uses: redhat-actions/podman-login@v1
         with:
           username: ${{ env.REGISTRY_USER }}
           password: ${{ env.REGISTRY_PASSWORD }}
           registry: ${{ env.IMAGE_REGISTRY }}
       - run: |
-          printf -v image_string '%s/%s:%s' \
-            ${{ env.IMAGE_REGISTRY }} \
-            ${{ env.IMAGE_NAMESPACE }} \
-            ${{ env.IMAGE_TAG }} ;
-          printf -v bundle_path '%s/%s' $GITHUB_WORKSPACE ${{ env.BUNDLE }} ;
-          tkn bundle push -f ${bundle_path} ${image_string} ;
-          tkn bundle list -o yaml --no-cache ${image_string} ;
\ No newline at end of file
+          for dir in ${GITHUB_WORKSPACE}/*
+          do
+            [[ -d "$dir" ]] || continue
+            printf 'Found Bundle dir: %s\n' $dir
+            BUNDLE_NAME="${dir##*/}"
+            BUNDLE_FILES=()
+            for file in  $dir/*.y{,a}ml
```

Comment:
It can be `*.yaml` or `*.yml` or even `*.json`
I always do "yaml", but I don't judge... some lazy people write "yml" and I'm not prejudice. :laughing: (read facetiously)
In all seriousness this code probably needs to be replaced with something else capable of only using new or modified files.

Reply:
Let's have a clean repo. We can add a check for that if we want and fail the ci if we found `yml` instead of `yaml`. And `json` shouldn't be allowed (or should be ignored) as we have a yaml linter. Maybe it makes sense to forbid anything that is not `yaml` or `md`.